proposals/0026-hlsl-long-vector-type.md

```diff
@@ -118,6 +118,13 @@ myBuffer.Store< vector<T, N> >(StartoffsetInBytes + 100, val);
 
 ```
 
+Long Vectors initialized from or dumped to a Raw Buffer must comply with the
+alignment rules of the underlying buffer. Therefore a vector `Load`ed or `Store`d
+to a `ByteAddressBuffer` must either naturally align to or be padded to 4 bytes.
```

Comment:
4 bytes seems possibly wrong. Should 64-bit types require 8-byte alignment? What about 16-bit types?

Reply:
Maybe I'm reading the docs wrong?

> The byte value must be a multiple of four so that it is DWORD aligned

https://learn.microsoft.com/en-us/windows/win32/direct3d11/direct3d-11-advanced-stages-cs-resources#byte-address-buffer

It explicitly says multiples of four, but also looks like DWORD is always 32 bits/4 bytes on windows

Reply:
Also, I'm not sure I understand how a 64 or 16 bit type comes into play for a byte address buffer.

It's just a buffer of bytes and the requirement is that the offset into the buffer be 4-byte aligned. A vector holding a 64-bit type will naturally fall into the 4-byte alignment and a single 16-bit value _will_ need to 4-byte align the offset into the buffer. (though a `vector<half,  2>` will naturally align)

Reply:
> This suggests that for longvectors, the alignment for the load should be the size of the scalar element type (2B for 16, 8B for 64...)

Ah yep, that makes a lot of sense. The `The byte value must be a multiple of four` is discussing the `uint` overloads of the function, not the `T` overload. I'll update the wording!